x-pack/filebeat/tests/system/test_http_endpoint.py

```diff
@@ -1,3 +1,4 @@
+from filebeat import BaseTest
```

Comment:
I guess this and the other python file were format changes performed by your IDE, could you remove them?